static/app/views/replays/replays.tsx

```diff
@@ -51,52 +83,69 @@ function Replays({location}: Props) {
     eventView,
   });
 
+  const shouldShowOnboardingPanel = useShouldShowOnboardingPanel();
+
   return (
     <Fragment>
-      <StyledPageHeader>
-        <HeaderTitle>
-          {t('Replays')} <ReplaysFeatureBadge space={1} />
-        </HeaderTitle>
-      </StyledPageHeader>
+      <Layout.Header>
+        <StyledLayoutHeaderContent>
+          <StyledHeading>
+            {t('Replays')} <ReplaysFeatureBadge space={1} />
+          </StyledHeading>
+        </StyledLayoutHeaderContent>
+      </Layout.Header>
```

Comment:
touched the header to make the spacing more like Profiling. Checkout the visual snapshot diffs.